### ISSUE TITLE
Charts for timeseries data

### ISSUE THREAD

**Issue by jh-RLI** [rl-institut/WAM_APP_FRED#57]
- Create charts using [this](https://github.com/kyper-data/python-highcharts)  Python translation layer for [Highcharts](https://www.highcharts.com/docs/getting-started/your-first-chart) javascript bib.

- use model_draft.openfred_timeseries_ch as datasource

- [ ] chreate openfred_ts_higchart and options
- [ ] create view returning Highchart based on userinput (time, technology)
- [ ] embed chart in html/js

**Comment by christian-rli** [comment]
When I try to run this branch I get the following error:
```
sqlalchemy.exc.ArgumentError: Mapper Mapper|OpenFredTimesSeries|openfred_timeseries_ch 
could not assemble any primary key columns for mapped table 'openfred_timeseries_ch'
```
and django fails. Is it my setup or is this supposed to not work yet?

**Comment by jh-RLI** [comment]
I will have a look at this - should work in my opinion.

**Comment by Bachibouzouk** [comment]
I can use a plotly graph :)

**Comment by jh-RLI** [comment]
okay that might be more easy for you :) @Bachibouzouk

@christian-rli there is no primary key column defined in the openfred_timeseries_ch  table

**Comment by christian-rli** [comment]
Is that necessary for importing it? I'll try to set it.

**Comment by jh-RLI** [comment]
seems like sqlalchemy orm is requiring it. A normalized db table should always have a primary key column.

**Comment by christian-rli** [comment]
column 'index' is now the primary key.

**Comment by jh-RLI** [comment]
this fixed the error :)

**Comment by Bachibouzouk** [comment]
This issue is now closed by #55
